src/components/Checkout.vue

```diff
@@ -25,16 +25,29 @@
       <div class="p-col-12">
         <div class="p-grid p-jc-between p-text-right p-ai-center">
           <div class="p-col-7 p-lg-9 p-pr-0">小計</div>
-          <div class="p-col-5 p-lg-3">$ {{ subtotal }}</div>
+          <div class="p-col-5 p-lg-3">$ {{ price_details.subtotal }}</div>
 
           <div class="p-col-7 p-lg-9 p-pr-0">運費</div>
-          <div class="p-col-5 p-lg-3">$ 0</div>
+          <div class="p-col-5 p-lg-3">$ {{ price_details.shipping_fee }}</div>
+
+          <div
+            v-if="price_details.free_shipping === 'true'"
```

Comment:
price_details.free_shipping 應該不是字串？

```suggestion
            v-if="price_details.free_shipping === true"
```

Reply:
改好了~
一起放在 6 項裡面了~